service/kinesis/api.go

```diff
@@ -7379,6 +7379,8 @@ type SubscribeToShardEventStream struct {
 // may result in resource leaks.
 func (es *SubscribeToShardEventStream) Close() (err error) {
 	es.Reader.Close()
+	es.StreamCloser.Close()
```

Comment:
How do we handle the error, if there is one when trying to close a Reader or StreamCloser?